pynest/nest/lib/hl_api_simulation.py

```diff
@@ -366,27 +378,53 @@ def SetKernelStatus(params):
         are counted, not spikes generated by devices such as ``poisson_generator``.
 
 
-    **Miscellaneous**
-
-    Parameters
-    ----------
-
-    dict_miss_is_error : bool
-        Whether missed dictionary entries are treated as errors
-    keep_source_table : bool
-        Whether to keep source table after connection setup is complete
-
     See Also
     --------
 
     GetKernelStatus
 
     """
+    # Resolve if missing entries should raise errors
+    raise_errors = params.get('dict_miss_is_error')
+    if raise_errors is None:
+        raise_errors = nest.GetKernelStatus('dict_miss_is_error')
```

Comment:
This needs to be just `GetKernelStatus`, otherwise you get `NameError: name 'nest' is not defined`

Reply:
one

Reply:
Let's see what the CI says ;) 🤞